src/constants.ts

```diff
@@ -59,7 +59,7 @@ export function getContext(network: string, publicKey?: string, privateKeyHex?:
   return context(getConfig(network), publicKey, privateKeyHex, privateKeyCB58)
 }
 
-function getConfig(network: string): NetworkConfig {
+export function getConfig(network: string | undefined): NetworkConfig {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums